packages/temen/src/isEqualWith/index.ts

```diff
@@ -0,0 +1,45 @@
+import { isEqual } from '../isEqual';
+
+/**
+ *
+ * 이 메서드는 `isEqual`와 동일하지만 `customizer`를 전달합니다.
+ * `customizer`가 `undefined`를 반환하면 비교는 메서드에 의해 처리됩니다.
+ * `customizer`는 최대 여섯 개의 인수를 전달합니다.
+ * (objValue, othValue [, index|key, object, other, stack])
+ *
+ * @param {any} value 비교할 값
+ * @param {any} other value와 비교할 값
+ * @param {Function} [customizer] 비교 조건
+ * @returns {boolean} Returns value가 other와 동일한지 여부
+ * @example
+ *
+ * function isGreeting(value) {
+ *   return /^h(?:i|ello)$/.test(value)
+ * }
+ *
+ * function customizer(objValue, othValue) {
+ *   if (isGreeting(objValue) && isGreeting(othValue)) {
+ *     return true
+ *   }
+ * }
+ *
+ * const array = ['hello', 'goodbye']
+ * const other = ['hi', 'goodbye']
+ *
+ * isEqualWith(array, other, customizer)
+ * // => true
+ */
+
+function isEqualWith<V, O>(
+  value: V,
+  other: O,
+  customizer: (value: V, other: O, key?: any, object?: any, otherObject?: any, stack?: any) => any
```

Comment:
```suggestion
  customizer: (value: V, other: O) => any
```

구현을 보면 이 두 개의 인자들만 사용되는데, 뒤에 있는 친구들은 언제 사용되는건가유??
뭔가 생긴 모양새를 보면 `customizer`를 재귀호출 해줘야 할 것 같은 모양새인데, 결국 이 재귀를 `isEqualWith` 외부에서 구현해야하는 상황인 것 같습니다.

차라리 그냥 `value, other` 두 인자만 받고 "이 두 개가 같니?"라는 구현은 유저한테 맡기는 것이 더 추상적인 설계인 것 같다는 생각도 들어요!

Reply:
요부분은 무지성 코딩을 해부렸네요! 뒤에 친구들도 같이 제외시켰습니다 https://github.com/Lubycon/lubycon-frontend-libraries/pull/128/commits/537a3f4593c0d4223bb2a443e938164c13cfa464